apps/web/src/views/PositionManagers/components/SingleTokenWarning.tsx

```diff
@@ -0,0 +1,30 @@
+import { useTranslation } from '@pancakeswap/localization'
+import { Message, MessageText, Text, Link } from '@pancakeswap/uikit'
+
+export const SingleTokenWarning = () => {
+  const { t } = useTranslation()
+
+  return (
+    <Message variant="primary" mt="15px">
+      <MessageText>
+        <Text fontSize={14} as="span" color="secondary">
+          {t(
+            'Single token deposits only. The final position may consist with both tokens. Learn more about the strategy',
+          )}
+        </Text>
+        <Link
+          bold
+          external
+          m="0 4px"
+          fontSize={14}
+          color="secondary"
+          display="inline-block !important"
+          href="https://www.ichi.org/"
```

Comment:
Update this before merge
```suggestion
          href="https://www.ichi.org/"
```